src/printer.js

```diff
@@ -1202,7 +1202,7 @@ function genericPrintNoParens(path, options, print) {
         ) {
           const value = n.value.extra ? n.value.extra.raw : n.value.raw;
           res = '"' +
-            value.slice(1, value.length - 1).replace(/"/g, "&quot;") +
+            value.slice(1, -1).replace(/"/g, "&quot;") +
```

Comment:
I know this unrelated, but I felt like simplifying it was definitely better and makes it more readable.